tips/glossary.md

```diff
@@ -14,6 +14,29 @@ window.addEventListener("click", function () {
 });
 ```
 
+### Frequently Used Terms Definitions 
+**git**: an open source, distributed version-control system
+
+**GitHub**: a platform for hosting and collaborating on Git repositories
+
+**commit**: a Git object, a snapshot of your entire repository compressed into a SHA
+
+**branch**: a lightweight movable pointer to a commit
```

Comment:
I would not describe a branch like this cause a branch is a snapshot of the code in which multiple commits can be added etc.

Reply:
branch :  is a snapshot of the code in which multiple commits can be added and get merged to the main branch 

what do you think ?

Reply:
ok git documentation also refers to it as pointer to a commit :D (https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell)

Reply:
technically, they're not wrong ; practically, we can explain it in more easily xD

Reply:
you can choose what you want to write :)